### ISSUE TITLE
A Critical Analysis of Image-based Camera Pose Estimation Techniques. (arXiv:2201.05816v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#14922]
A Critical Analysis of Image-based Camera Pose Estimation Techniques. (arXiv:2201.05816v1 [cs.CV])<br>
https://ift.tt/33ojBF3<br>
<p>Camera, and associated with its objects within the field of view, localization could benefit many computer vision fields, such as autonomous driving, robot navigation, and augmented reality (AR). In this survey, we first introduce specific application areas and the evaluation metrics for camera localization pose according to different sub-tasks (learning-based 2D-2D task, feature-based 2D-3D task, and 3D-3D task). Then, we review common methods for structure-based camera pose estimation approaches, absolute pose regression and relative pose regression approaches by critically modelling the methods to inspire further improvements in their algorithms such as loss functions, neural network structures. Furthermore, we summarise what are the popular datasets used for camera localization and compare the quantitative and qualitative results of these methods with detailed performance metrics. Finally, we discuss future research possibilities and applications.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
